tardis/model.py

```diff
@@ -268,6 +268,16 @@ def to_hdf(self, path_or_buf, path='', plasma_properties=None):
         None
 
         """
+        properties = ['t_inner', 'ws', 't_rads', 'v_inner', 'v_outer']
+        for p in properties:
+            output_value = getattr(self, p)
+            if hasattr(output_value, 'si'):
+                output_value = output_value.si.value
+            if np.isscalar(output_value):
+                output_value = [output_value]
+            data = pd.DataFrame(output_value)
```

Comment:
Are they exactly the same length? I don't know but it's not important. If
one element is longer than the rest the others get filled with np.nan. One
would have to take care of that when reading the valus, that's all


Fotis Tsamis <notifications@github.com> schrieb am Mo., 20. Juni 2016 um
17:18 Uhr:

> In tardis/model.py
> <https://github.com/tardis-sn/tardis/pull/589#discussion_r67708417>:
>
> > @@ -268,6 +268,16 @@ def to_hdf(self, path_or_buf, path='', plasma_properties=None):
> >          None
> >
> >          """
> > +        properties = ['t_inner', 'ws', 't_rads', 'v_inner', 'v_outer']
> > +        for p in properties:
> > +            output_value = getattr(self, p)
> > +            if hasattr(output_value, 'si'):
> > +                output_value = output_value.si.value
> > +            if np.isscalar(output_value):
> > +                output_value = [output_value]
> > +            data = pd.DataFrame(output_value)
>
> Yeah, I hadn't noticed that they are all the same length. Is that always
> the case? Because otherwise this won't work.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/tardis-sn/tardis/pull/589/files/b9ca672ee6796a356ca4593f1e8886adfb82397a..d8f3a53a09090c6946f8f366ae5e376fb6e68b4c#r67708417>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF4OjmRNXUTretIFxgKxUjopvBLYHhu0ks5qNq9DgaJpZM4I1pWa>
> .
>